gmm.py

```diff
@@ -331,3 +370,56 @@ def __update_pi(self, pi):
         assert pi.size() in [(1, self.n_components, 1)], "Input pi does not have required tensor dimensions (%i, %i, %i)" % (1, self.n_components, 1)
 
         self.pi.data = pi
+
+def get_kmeans_mu(X, K, min_delta=1e-3):
```

Comment:
Looks good! Might make sense to use `x` as the variable name (as done earlier), and maybe `n_centers` to indicate what `K` is used as.

Reply:
I have moved this function into the class and add  `init_params `. To prevent the mu overflow in calculate, I add min\max normalization in this function. The judge of nan in `fit` doesn't work since nan can't bu judged by operater "=="